pundix/assetlist.json

```diff
@@ -0,0 +1,87 @@
+{
+  "$schema": "../assetlist.schema.json",
+  "chain_name": "Pundi X Chain",
+  "assets": [
+    {
+      "description": "Pundi X Token",
+      "denom_units": [
+        {
+          "denom": "ibc/55367b7b6572631b78a93c66ef9fdfce87cde372cc4ed7848da78c1eb1dcdd78",
+          "exponent": 0
+        },
+        {
+          "denom": "PUNDIX",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/55367b7b6572631b78a93c66ef9fdfce87cde372cc4ed7848da78c1eb1dcdd78",
+      "name": "Pundi X Token",
+      "display": "PUNDIX",
+      "symbol": "PUNDIX",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/pundi-x-token-logo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/pundi-x-token-logo.svg"
+      },
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "fxcore",
+            "base_denom": "eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38",
+            "channel_id": "channel-0"
+          },
+          "chain": {
+            "channel_id": "channel-0",
+            "path": "transfer/channel-0/eth0x0FD10b9899882a6f2fcb5c371E17e70FdEe00C38"
+          }
+        }
+      ],
+      "coingecko_id": "pundi-x-2",
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/pundi-x-token-logo.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/pundi-x-token-logo.svg"
+        }
+      ]
+    },
+    {
+      "description": "PURSE Token",
+      "denom_units": [
+        {
+          "denom": "bsc0x29a63f4b209c29b4dc47f06ffa896f32667dad2c",
+          "exponent": 0
+        },
+        {
+          "denom": "PURSE",
+          "exponent": 18
+        }
+      ],
+      "base": "bsc0x29a63f4b209c29b4dc47f06ffa896f32667dad2c",
+      "name": "PURSE Token",
+      "display": "PURSE",
+      "symbol": "PURSE",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/purse-token-logo.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/pundix/images/purse-token-logo.svg"
+      },
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "binance smart chain",
+            "base_denom": "0x29a63F4B209C29B4DC47f06FFA896F32667DAD2C",
```

Comment:
Please add this token to BSC (at [/_non-cosmos/binancesmartchain/assetlist.json](https://github.com/cosmos/chain-registry/blob/master/_non-cosmos/binancesmartchain/assetlist.json))